install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# This is an install script for notes
+
+function assertInstalled() {
```

Comment:
Figured it out. It's because we're doing a switch to root, and the function is being curled as regular user. if you do that curl under root, it doesn't happen.